graylog2-server/src/main/java/org/graylog2/inputs/transports/AmqpConsumer.java

```diff
@@ -217,19 +265,7 @@ public void read(Command inboundCommand) {
             factory.setUsername(username);
             factory.setPassword(encryptedValueService.decrypt(password));
         }
-        connection = factory.newConnection();
-        channel = connection.createChannel();
-
-        if (null == channel) {
-            LOG.error("No channel descriptor available!");
-        }
-
-        if (null != channel && prefetchCount > 0) {
-            channel.basicQos(prefetchCount);
-
-            LOG.debug("AMQP prefetch count overriden to <{}>.", prefetchCount);
-        }
-
+        connection = factory.newConnection(f("graylog-node-%s-input%s-%s", nodeId.getNodeId(), sourceInput.toIdentifier(), hashCode()));
```

Comment:
maybe add a short comment explaining why we add `hashCode()` to the string here

Reply:
Done